docs/rabbitmqctl.1.xml

```diff
@@ -1237,14 +1235,51 @@
 
       <variablelist>
         <varlistentry role="usage-has-option-list">
-          <term><cmdsynopsis><command>list_queues</command> <arg choice="opt">-p <replaceable>vhost</replaceable></arg> <arg choice="opt" role="usage-option-list"><replaceable>queueinfoitem</replaceable> ...</arg></cmdsynopsis></term>
+          <term>
+            <cmdsynopsis><command>list_queues</command> <arg choice="opt">-p <replaceable>vhost</replaceable></arg> <group choice="opt"><arg>--offline</arg><arg>--online</arg><arg>--local</arg></group> <arg choice="opt" role="usage-option-list"><replaceable>queueinfoitem</replaceable> ...</arg></cmdsynopsis>
+          </term>
           <listitem>
             <para>
               Returns queue details. Queue details of the <command>/</command> virtual host
               are returned if the "-p" flag is absent. The "-p" flag can be used to
               override this default.
             </para>
             <para>
+              Displayed queues can be filtered by their status or
```

Comment:
`offline`/`online`/`local` are not strictly `queueinfoitems`, but are listed as such in the help. What if we add those details just below [usage](https://github.com/rabbitmq/rabbitmq-server/pull/911/files#diff-b96d4e7ccd3d984656b8ab3534e6b460R1230) ?

```
  list_queues [-p <vhost>] [--offline|--online|--local] [<queueinfoitem> ...]
...
<queueinfoitem> must be a member of the list [[--offline], [--online],
[--local], name, durable, auto_delete, arguments, policy, pid, owner_pid,
exclusive, exclusive_consumer_pid, exclusive_consumer_tag, messages_ready,
```